### ISSUE TITLE
support disable fixTwoCNChar check in button

### ISSUE THREAD

**Issue by whtsky** [ant-design/ant-design#10188]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
https://github.com/ant-design/ant-design/blob/master/components/button/button.tsx#L98
fixTwoCNChar Check in button is causing performance issue when putting a lot of button on the same page. And our designer doesn’t enforce space between two Chinese char.

### What does the proposed API look like?
add `disableFixTwoCNCharCheck` prop to Button component

**Comment by wmzhong** [comment]
Is this accepted to add the `disableFixTwoCNCharCheck` prop?

**Comment by stonehank** [comment]
May be can use like '<Button><span>点</span>击</Button>' to avoid fixTwoCNChar check...so ugly....
but if have a lot of Button, it's not a good choice

**Comment by whtsky** [comment]
@stonehank this solution requires hardcode text so it's hard to do i18n with it :(

**Comment by stonehank** [comment]
@whtsky Indeed, the issue had opened more than one month early, may be you can try PR~
